core/vm/lfvm/converter.go

```diff
@@ -22,6 +22,7 @@ type cache_val struct {
 var changedAddress01 = common.HexToAddress("0xA7CC236F81b04c1058e9bfb70E0Ee9940e271676")
 var changedAddress02 = common.HexToAddress("0xAD0FB83a110c3694faDa81e8B396716a610c4030")
 var changedAddress03 = common.HexToAddress("0xA8B3C9f298877dD93F30E8Ed359956faE10E8797")
+var changedAddress04 = common.HexToAddress("0x6DBd5d37397afF80BE434F74cc89b9d933635784")
```

Comment:
Do mainnet and testnet share the same list of changed addresses?

Reply:
Yes, they share the same list, but it is only for the converted code cache, so it doesn't matter that it is for mainnet or testnet.